hack/testing-manifests/wandb/wandb-dev-v2.yaml

```
@@ -34,3 +37,4 @@ spec:
     managedClickhouse:
       telemetry:
         enabled: true
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)